### PR TITLE
update button implementation

### PR DESCRIPTION
**Pull request by tballmsft**

### CODE REVIEW

**Comment by tballmsft** [comment]
I thought we agreed the button would be ms (not us).

**Comment by mmoskal** [comment]
Correct, ms. Which is why I was asking why us ;)

File: services/button.c [hunk 1]
```diff
@@ -34,22 +26,20 @@ static void update(srv_t *state) {
...
     }
 
     if (state->pressed) {
         uint32_t presslen = now - state->press_time;
-        if (presslen >= state->click_hold_time * 1000 &&
-            state->prev_presslen < state->click_hold_time * 1000)
-            jd_send_event(state, JD_BUTTON_EV_HOLD);
-        state->prev_presslen = presslen;
+        if (presslen >= state->next_hold) {
+            state->next_hold = presslen + 500000;
```
Comment (mmoskal) [inline services/button.c:39]: I would do next_press += 500000 to avoid any drift but either way is fine

**Review by mmoskal — APPROVED** [review]

File: services/button.c (hunk 1 above)
Comment (mmoskal) [reply services/button.c:39]: Next hold, sorry

**Pull request merged**